### PR TITLE
Add 184-elastic collisions challenge

### PR DESCRIPTION
**Pull request by kfahn22**
This is still missing:

- index image
- description
- videoID
- link to Dipam's paper
- link to other paper mentioned

Date is currently set to today--will need to be updated.

### CODE REVIEW

**Comment by dipamsen** [comment]
2d collisions article on vobarian.com: https://www.vobarian.com/collisions/2dcollisions2.pdf

My derivation/explanation of the collision formulae: https://dipamsen.github.io/notebook/collisions.pdf

**Comment by kfahn22** [comment]
@dipamsen Thanks for the links!  I have added the link to the NOC website, but kept the link for preorder. @shiffman you can decide whether to delete.

File: content/videos/challenges/184-elastic-collisions/index.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
+  "title": "Elastic Collisions",
+  "videoNumber": "184",
+  "description": "",
+  "videoId": "",
+  "nebulaSlug": "codingtrain-coding-challenge-184-elastic-collisions",
+  "date": "2024-07-12",
+  "languages": ["p5.js", "JavaScript"],
+  "topics": ["elastic collisions", "physic's simulation"],
```
Comment (fturmel) [inline content/videos/challenges/184-elastic-collisions/index.json:9]: tiny tweak: "physics simulation", it should not be possessive

**Comment by shiffman** [comment]
I've added a description, index image, and made other small changes. @dipamsen I think it works to have both the main URL and the no starch page as the links in the reference. I was hoping to have a way to order direct from natureofcode.com by now (I'm in touch with the warehouse/distributor on a way to integrate a shopify cart from noc.com itself) but I don't have all the pieces yet and I don't want to wait to release....

**Comment by shiffman** [comment]
The video is now members only I'll make public tomorrow Saturday July 13. Merging but welcome additional fixes and edits in new PRs!

**Pull request merged**